python/py-fqdn/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-fqdn
+version             1.5.1
+revision            0
+categories-append   devel
+license             MPL-2
+supported_archs     noarch
```

Comment:
it the package does not depend on the architecture and it installs the same files on all macOS versions, which typically can be checked from PyPI if the downloadable .whl files are names `*none-any.whl`.
In that case, you should also set `platforms {darwin any}`